frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.sass

```diff
@@ -3,3 +3,9 @@ ng-select.wp-inline-create--reference-autocompleter
 
   .ng-clear-wrapper
     display: none
+    
+.ng-select.ng-select-single .ng-select-container .ng-value-container .ng-input 
```

Comment:
This selector is applied everywhere within the application, and thus for every autocompleter that we have, not only the 'parent relation'-autocompleter. Without checking it in detail, I assume that it might break on other places. So please scope the selector to the class `wp-inline-create--reference-autocompleter` which is already used at the top of the file. This has also the advantage that you don't have to specify the selector that much.